### PR TITLE
Feature/custom payment

### PR DESCRIPTION
**Pull request by billfeng**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.0.13",
     "react-stripe-checkout": "^2.6.3",
```
Comment (kumilange) [inline package.json]: @bill-feng Using checkout as well?

File: src/styles/Pay.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* #payment-info-table {
+  width: 100%;
+} */
```
Comment (kumilange) [inline src/styles/Pay.css]: @bill-feng Let's remove this comment out :)

File: src/components/Event.js [hunk 3]
```diff
@@ -88,22 +88,12 @@ class Event extends Component {
...
-                  stripeKey="pk_test_6pRNASCoBOKtIshFeQd4XMUh"
-                  currency="JPY"
-                  name="Live Jazz Co."
-                  image="../logo.png"
-                  description={`1 ticket for ${this.props.event.name}`}
-                  amount={2000}
+              <Link to={'/pay'}>
+                <RaisedButton
+                  className="mui-button"
+                  label="Reserve"
```
Comment (kumilange) [inline src/components/Event.js]: @bill-feng Seems like the primary color for reserve button is gone(white now) :( Do you think you can put the style back?

**Comment by kumilange** [comment]
@bill-feng Great job, it was fast:) I left some minor comments but other than that, LGTM! 👍

**Review by kumilange — DISMISSED** [review]

File: src/components/Event.js (hunk 3 above)
Comment (billfeng) [reply src/components/Event.js]: Yea the whole button is gone now. We need to do styling separately if we want!

File: src/styles/Pay.css (hunk 2 above)
Comment (billfeng) [reply src/styles/Pay.css]: Fixed!

File: package.json (hunk 1 above)
Comment (billfeng) [reply package.json]: Fixed!

**Review by kumilange — APPROVED** [review]

**Pull request merged**